justfile

```diff
@@ -47,8 +47,6 @@ print-staging-report: (_run_ops_bin 'print_staging_report')
 
 check-genesis-integrity: (_run_ops_bin 'check_genesis_integrity')
 
-codegen: (_run_ops_bin 'codegen')
```

Comment:
why was this deleted?

Reply:
Because you now are required to provide `--l1-rpc-urls` flag for `codegen` therefore I think its cleaner to invoke the `codegen` go program directly instead of via `just` recipe